app/models/event/festival.rb

```diff
@@ -0,0 +1,7 @@
+#  Copyright (c) 2012-2019, Schweizer Blasmusikverband. This file is part of
```

Comment:
```suggestion
#  Copyright (c) 2019, Schweizer Blasmusikverband. This file is part of
```

This only exists since this year, right? As long as we manually adapt those license-blocks, we can be accurate.